src/lib/openapi/spec/client-feature-schema.ts

```diff
@@ -9,55 +9,67 @@ export const clientFeatureSchema = {
     $id: '#/components/schemas/clientFeatureSchema',
     type: 'object',
     required: ['name', 'enabled'],
+    description:
+        'Feature toggle configuration used by SDKs to evaluate state of a toggle',
     additionalProperties: false,
     properties: {
         name: {
             type: 'string',
+            description:
+                'The unique name of a feature toggle. Is validated to be URL safe on creation',
+            example: 'new.payment.flow.stripe',
         },
         type: {
             type: 'string',
+            description: 'What kind of feature flag is this',
+            example: 'release',
         },
         description: {
             type: 'string',
+            description: 'A description of the toggle',
             nullable: true,
-        },
-        createdAt: {
-            type: 'string',
-            format: 'date-time',
-            nullable: true,
-        },
-        lastSeenAt: {
-            type: 'string',
-            format: 'date-time',
-            nullable: true,
+            example: 'No variants here',
         },
         enabled: {
             type: 'boolean',
+            description:
+                'Whether the feature flag is enabled for the current API key or not. This is ANDed with the evaluation results of the strategies list, so if this is false, the evaluation result will always be false',
+            example: true,
         },
         stale: {
+            description:
+                'If this is true Unleash believes this feature toggle has been active longer than Unleash expects a toggle of this type to be active',
```

Comment:
Doesn't this have to be explicitly set by the users? I thought Unleash only said features *may* be stale, but that it required a user to mark them as such?

```suggestion
                'Whether this feature toggle has become [stale](https://docs.getunleash.io/reference/technical-debt#stale-and-potentially-stale-toggles "Stale toggles") or not.',
```